streaming.md

```diff
@@ -138,47 +210,53 @@ FlinkPravegaWriter<MyClass> pravegaSink = FlinkPravegaWriter.<MyClass>builder()
 DataStream<MyClass> stream = ...
 stream.addSink(pravegaSink);
 ```
-
-### Parameters
-A builder API is provided to construct an instance of `FlinkPravegaWriter`. See the table below for a summary of builder properties.  Note that the builder accepts an instance of `PravegaConfig` for common configuration properties.  See the [configurations](configurations) page for more information.
-
-|Method                |Description|
-|----------------------|-----------------------------------------------------------------------|
-|`withPravegaConfig`|The Pravega client configuration, which includes connection info, security info, and a default scope.|
-|`forStream`|The stream to be written to.|
-|`withWriterMode`|The writer mode to provide best-effort, at-least-once, or exactly-once guarantees.|
-|`withTxnLeaseRenewalPeriod`|The transaction lease renewal period that supports the exactly-once writer mode.|
-|`withSerializationSchema`|The serialization schema which describes how to turn events into byte messages.|
-|`withEventRouter`|The router function which determines the routing key for a given event.|
-|`enableMetrics`|true or false to enable/disable reporting Pravega reader group metrics. By default, the metrics option is enabled.|
+## Serialization
+Please, see the [serialization](serialization) page for more information on how to use the serializer.
 
 ### Parallelism
-The data sink supports parallelization.  Use the `setParallelism` method to configure the number of parallel instances to execute.
+FlinkPravegaWriter supports parallelization. For more information please revist the concept [Parallelism](#parallelism) discussed in the above section.
 
 ### Event Routing
-Every event written to a Pravega stream has an associated routing key.  The routing key is the basis for event ordering.  See the [Pravega documentation](http://pravega.io/docs/pravega-concepts/#events) for details.
+Every event written to a Pravega stream has an associated Routing Key.  The Routing Key is the basis for event ordering.  See the [Pravega documentation](http://pravega.io/docs/pravega-concepts/#events) for details.
 
-To establish the routing key for each event, provide an implementation of `io.pravega.connectors.flink.PravegaEventRouter` when constructing the writer.
+When constructing the FlinkPravegaWriter, to establish the Routing Key for each event, provide the below mentioned implementation:
+```
+io.pravega.connectors.flink.PravegaEventRouter` 
+```

```

Comment:
Can we change it to like below?

When constructing the `FlinkPravegaWriter`, please provide an implementation of `io.pravega.connectors.flink.PravegaEventRouter` which will guarantee the event ordering. In Pravega, events are guaranteed to be ordered at the segment level.

Reply:
Accepted suggestion